blueprints/ember-cli-blanket/files/tests/blanket-options.js

```diff
@@ -8,7 +8,8 @@ var options = {
   enableCoverage: true,
   cliOptions: {
     reporters: ['json']
-  }
+  },
+  autostart: true
```

Comment:
Any reason not to put this in cliOptions?  It's our hash for ember-cli-blanket options. 

What if blanket comes along with auto start and clobbers our meaning.